swap_meet/vendor.py

```diff
@@ -1,2 +1,105 @@
+
+from .item import Item
+
+
+
+# WAVE 1
 class Vendor:
-    pass
+    # constructor that is by default empty, but checks for input
+    def __init__(self, inventory = [] ):
+        if inventory :
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+    # method thats adds an item to inventory
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    # method that removes an item from inventory
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
We can reverse the logic here and use a guard clause here to eliminate the syntax that the `else` adds:

```suggestion
    def remove(self, item):
        if item not in self.inventory:
            return False
        self.inventory.remove(item)
        return item

```